packages/react-test-renderer/src/ReactTestRendererScheduling.js

```diff
@@ -12,7 +12,7 @@ import type {Deadline} from 'react-reconciler/src/ReactFiberScheduler';
 // Current virtual time
 export let nowImplementation = () => 0;
 export let scheduledCallback: ((deadline: Deadline) => mixed) | null = null;
-export let yieldedValues: Array<mixed> | null = null;
+export let yieldedValues: Array<mixed> = [];
```

Comment:
The transient `null` value didn't seem to be buying us anything, and it complicated the logic below– so I ditched it.